admin/class-auction-software-admin.php

```diff
@@ -1036,8 +1036,13 @@ public function auction_software_auctions_list_endpoint() {
 					} elseif ( false === $product->is_started( $product->get_id() ) ) {
 						$content .= '<td >--</td><td>--</td>';
 					} elseif ( ! $product->is_ended( $product->get_id() ) ) {
-						$content .= '<td ><span class="auctiontime-left timeLeft' . $product->get_id() . '"></span></td>
-                                        <td><a href="http://localhost/wooauction/?product=' . get_the_title() . '" data-quantity="1"
+                        $date_to_or_from = $product->get_auction_date_to();
+                        $content .= '<td ><span class="auctiontime-left timeLeft' . $product->get_id() . '"></span>
+                                          <input type="hidden" class="timeLeftId" name="timeLeftId" value="' . $product->get_id() . '">
+                                          <input type="hidden" class="timeLeftValue'. $product->get_id() .'"  value="' . $date_to_or_from . '" />
+
+                                     </td>
+                                        <td><a href="' . get_permalink() . '" data-quantity="1"
                                         class="button product_type_auction_simple add_to_cart_button" data-product_id="' . $product->get_id() . '"
```

Comment:
class names added are specific to simple auction, should be generic as we have three auctions - simple, reverse, and penny.